### PR TITLE
chore: use ICurrency import from request-logic-types instead of dist

### PR DESCRIPTION
**Pull request by siibars**
## Description of the changes

Importing ICurrency from types/RequestLogicTypes so that further extensions can be supported

### CODE REVIEW

**Review by yomarion — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -3289,6 +3289,33 @@
   resolved "https://registry.yarnpkg.com/@redocly/react-dropdown-aria/-/react-dropdown-aria-2.0.11.tgz#532b864b329237e646abe45d0f8edc923e77370a"
   integrity sha512-rmuSC2JFFl4DkPDdGVrmffT9KcbG2AB5jvhxPIrOc1dO9mHRMUUftQY35KZlvWqqSSqVn+AM+J9dhiTo1ZqR8A==
 
+"@requestnetwork/currency@0.2.0":
```
Comment (vrolland) [inline yarn.lock:3292]: This is odd. Let me check something.

Comment (vrolland) [reply yarn.lock:3292]: Found it. We made a publish before I merge my last PR. 

Can you wait the merge of: https://github.com/RequestNetwork/requestNetwork/pull/436.
Then pull the change from master, do a `yarn` and then commit the new `yarn.lock` please.

**Review by vrolland — APPROVED** [review]
Nice PR.
It is actually a fix but we can keep the denomination `chore:`  anyway.

Please wait for the merge of https://github.com/RequestNetwork/requestNetwork/pull/436 before merging this one.

File: yarn.lock (hunk 1 above)
Comment (vrolland) [reply yarn.lock:3292]: merged done.

**Comment by siibars** [comment]
Hi @vrolland 
The rebase with master has been done and the commit denomination is now a fix 😊

**Review by vrolland — APPROVED** [review]
Lgtm👍

**Pull request merged**